README.md

```diff
@@ -4,7 +4,7 @@ This project was made to test knowledge of html and css creating a form.
 
 # :writing_hand: Author
 
-This project was made by ALPANNA-GH.
+This project was made by [ALPANNA-GH](https://github.com/ALPANNA-GH).
 
```

Comment:
[Optional[ Consider adding how to set up and open the project in your local machine :computer:.

Reply:
- [ ] It is important when giving reviews to phrase your comments politely by using polite suggestions such as `I think it would be better to use semantic tags`, `I want to kindly suggest`, `please can you make this necessary change` etc
- [ ] You are also required to always include an explanation, a "why", for each requested change. for example `Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible`. 
- > Please kindly adjust your comments to meet these requirements 👍 ✨